src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/JQueryQueryStringValueProvider.cs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Microsoft.AspNetCore.Mvc.Internal;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding
+{
+    /// <summary>
+    /// An <see cref="IValueProvider"/> for jQuery formatted query string data.
+    /// </summary>
+    public class JQueryQueryStringValueProvider : BindingSourceValueProvider, IEnumerableValueProvider
```

Comment:
Why does this class exist? Looks identical to `JQueryFormValueProvider`.

If it must exist, should also duplicate `JQueryFormValueProviderTest`.

Reply:
This is important and unaddressed.

Reply:
Actually I did add the test?https://github.com/aspnet/Mvc/pull/7441/commits/503c2d2dfc41e9cf8bb58bcf6f86d7a1ec09ad9a#diff-d9872225e3146cb9b2bd227f6aaa7f98R10